pandas/core/accessor.py

```diff
@@ -282,7 +282,8 @@ def decorator(accessor):
 @doc(
     _register_accessor,
     klass="DataFrame",
-    others="register_series_accessor, register_index_accessor",
+    others=("register_series_accessor : Register a custom accessor on Series objects.\n"
+            "register_index_accessor : Register a custom accessor on Index objects."),
```

Comment:
Instead of doing this, I prefer to simply repeat the three methods in all docstrings. In each docstring we'll have a self-reference in the See Also, but that's not a big deal. Much better that all this complexity here.